Daily journal vs Remote Daily (#7)
Efesto/blog: _posts/2022-09-02-Daily-Journal.md

---
layout: post
title: "Remote daily and Daily journal"
---

> Recently I've started leading a small engineering team of 4 in Kloeckner.I.

Author: ```suggestion
Recently I've started leading a small engineering team of 4.
```

explain what leading means

This team was formed anew and in a remote working context and given the novelty of it, I took the occasion for rethinking some team-working approaches including the infamous remote daily.

> Like many people in my field, I've worked remotely since the beginning of 2020 and therefore I had a good amount of time for developing some ideas regarding the remote daily.

Author: why?

## Why remote daily sucks

> Remote daily requires people to interrupt whatever they are doing for attending a meeting to explain what they are doing.
> 
> ### Collaborative messaging systems
> I think it made a lot of sense when we didn't have collaborative messaging tools like Slack or MS Teams.

Author: ```suggestion
### Collaborative messaging systems
Dailies require people to interrupt whatever they are doing for attending a meeting to explain what they are doing.

I think it made a lot of sense when we didn't have collaborative messaging tools like Slack or MS Teams.
```

> Those tools allow us to stay connected constantly to a flow of messages from every side of an organization and to asynchronously communicate with whomever, asking for help, sending structured messages and so on.

Author: this can be refocused

> Given these premises, I don't get why having a synchronous meeting could be more effective than dropping a message somewhere.

Author: not my perspective

### Fear of missing a meeting
In a remote organization, it's easy to imagine that someone is not doing his part.

And because of that, it's even easier to imagine that someone else is thinking the same about us.

The statement "you don't have to attend a meeting if we don't see value in it" has a different meaning depending on the hygiene of workplace culture. 

> Somewhere and/or for someone a meeting can be seen as a way to report in, despite any good intention.

Author: ```suggestion
Somewhere and/or for someone a meeting can be perceived as a way to report in, despite any good intention.
```

add more explaination on why it's bad

And this one happens every day.

### Time management
People doing creative jobs know the importance of [flow](https://www.psychologytoday.com/us/basics/flow).

> A meeting is a sure way to prevent it, not only for the duration of the meeting but also before and sometime after.

Author: ```suggestion
By setting a meeting not only the flow is damaged by an interruption but it requires an additionally recovery time for regaining it.
```

> The idea for avoiding it would be of having the daily at the end or the beginning of the working day, but in a remote environment, the differences between each one's working day are even more marked.

Author: rephrase

### Lost information
Remote dailies usually are not recorded and the information transmitted there stays only between the sender and the recipient and only if the headphones/microphones worked fine and the recipient was giving attention

It could easily happen that people misunderstand or need to communicate information in different ways, having to rely on the previously mentioned messaging systems.

> Additionally, only the attendants to such meetings can benefit from it, leaving whoever wasn't able to attend in obscurity, needing to ask someone else to repeat.

Author: ...and more

> ## Still value in sync

Author: ```suggestion
## The value of synchronizing
```

> There's still value tho in having in a team a way to synchronize and keep visibility on what each other is doing.

Author: rephrase

Someone might be blocked on a problem that someone else can solve or that the first is working on the same thing as the second.

I wanted to find a way to let my team be able to do the job they are supposed to do without forcing on themselves any useless bureaucracy.

## Introducing Daily Journal

> In the past, I had the pleasure to work with an amazing eXtreme Programming team and there we used to journal and the end of our working day.

Author: ```suggestion
In the past, I had the pleasure to work with an amazing eXtreme Programming team that used to journal at the end of its working day.
```

> The idea was to be able to communicate with other people other than ourselves, Slack wasn't a tool back then and that tool on an internal wiki was working perfectly for the purpose.

Author: ```suggestion
We needed a way to communicate with other people other than ourselves, Slack didn't exist back then and publishing our journal on an internal wiki worked perfectly for that purpose.
```

> I've decided then to propose a similar tool to my new team in the form of a single, team-shared google document.

Author: ```suggestion
I've decided then to propose a similar tool to my new team in the form of a single, collaborative, Google document.
```

> Each team member was asked to fill at the beginning of each day by answering the following questions:

Author: ```suggestion
Each team member fills their entry at the beginning of each day by answering the following questions:
```

- What did I do yesterday?
- Am I blocked with anything?
- What will I do today?

The advantages of such an approach became quickly evident:

### Recording history

> Having a single, indexed document allows recovery of what happened on any day, simplifies radically the "recovery time" after a holiday or a sick leave and prevents losing information.

Author: ```suggestion
Having a single, searchable document allows recalling of what happened on any day, simplifies radically the "recovery time" after a holiday or a sick leave and prevents losing information.
```

Not being blended with a chat simplifies this mechanism without any real cost.

### Ordering ideas
Journaling is an excellent way for ordering ideas and the daily journal helps the writer and the reader as well.

### No need for "a quiet place"

> The daily journal doesn't require interrupting whatever one is doing or changing wherever one is working from just for having a status report.

Author: ```suggestion
The daily journal doesn't require interrupting whatever one is doing or changing wherever one is working from for connecting in a meeting.
```

> It helps keep the flow, which is an important thing that a team lead should defend.

Author: ```suggestion
It helps keeping the flow, which is an important thing that a team lead should defend.
```

### Allows more detailed information
By having a document dedicated to collecting daily insights, it can be also a space for triggering conversations on other channels and for sharing information.

Also, hyperlinks are pretty neat.

### Transparent
Being a shareable document means that whoever is interested in the state of the things of a team can simply open it at any moment and read the evolution of the work.

Our stakeholders weren't ever interested to such a degree in our internalities, but if necessary, the document is public and reachable from our channels.

## Our experience and conclusions
Since the very first days of adoption, the team preferred the daily journal a lot over the remote daily.

It helped us in quickly onboard our first new team member because it served as a tool for transmitting our team working culture.

To those who might think that avoiding having a daily meeting face-to-face might yield to team alienation, I can suggest focusing on different tools for keeping people connected other than a meeting conceived with the goal of status synchronization.

> In our case we have a private chat for engineers only where everyone is free to chat and write whatever might cross her mind, it's a nice way to vent out and keeps the team cohesive.

Author: ```suggestion
In our case we have a private chat for techs only where everyone is free to chat and write whatever might cross their mind, it's a nice way to vent out and keeps the team cohesive.
```
